docs/UserGuide.md

```diff
@@ -353,9 +385,10 @@ will discard all data and start with an empty data file at the next run.
 Shows the schedule of a specified person.
 
 Format: `viewSchedule INDEX`
-* Shows the schedule of a specific person at INDEX
+
+* Shows the schedule of a specific person at INDEX.
 * The index refers to the index number shown in the displayed person list.
-* The index must be positive. Eg. 1, 2, 3…
+* The index must be positive. Eg. 1, 2, 3….
```

Comment:
```suggestion
* The index **must be positive**. Eg. 1, 2, 3….
```

Reply:
Now that I look at it, I think examples should not end with fullstop either.